src/components/Paint.tsx

```diff
@@ -84,12 +84,9 @@ function Paint(props: PaintProps) {
 
     const storageHandler = (e: StorageEvent) => {
         if (e.key == 'stack' && !selfStore) {
-            debug('different instance wrote to local storage; locking');
+            debug('different instance wrote to local storage');
             setSelfStore(false);
             // setStack(JSON.parse(e.newValue) || []);
-            setCannotDraw(true);
-            setCanUndo(false);
-            setCanToggle(false);
```

Comment:
So this code is in place so that a single user can't open multiple instances of the website and draw from all of them--that's why the draw controls lock. That's what `selfStore` is for---if instance A detects that local storage has updated but `selfStore` is false, that means there's another instance B on the computer that drew a stroke, so instance A locks its controls.

Reply:
basically it kept getting locked for some reason.. im not entirely sure why.. im assuming the !selfStore wasn't being set properly?

Reply:
wait why is this function necessary.. the storage should be the same no?

Reply:
Hmm maybe, I'll take a look on my Macbook and see if I can figure anything out